src/runtime_src/core/pcie/emulation/common_em/config.cxx

```diff
@@ -980,4 +981,45 @@ namespace xclemulation{
     }
   }
 
+  int getIPName2Index(const char *name,const char* buffer)
```

Comment:
Copy-paste all my comments about this déjà vu code.
Fundamentally this looks like a typical DNRY red flag! :-(

Reply:
done